Marlin/src/HAL/HAL_LPC1768/arduino.cpp

```diff
@@ -85,6 +85,7 @@ extern "C" void delay(int msec) {
 
 // IO functions
 // As defined by Arduino INPUT(0x0), OUPUT(0x1), INPUT_PULLUP(0x2)
+#define OUPUT  0x1
```

Comment:
I appear to have a typo in the comment, should be OUTPUT, which is already defined in arduino.h